src/components/webView/webViewInstances.js

```diff
@@ -3,7 +3,7 @@ import { createIframe } from './iframe.web'
 
 export const TermsOfUse = createIframe(
   `https://community.gooddollar.org/${Config.isEToro ? 'pilot-terms' : 'dappterms'}/`,
-  'Terms of Use'
+  'Privacy Policy & Terms'
```

Comment:
@yaroslav-fedyshyn-nordwhale modify to the urls as given by @Talbatz in the other issue you mentioned

Reply:
done